pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -157,29 +157,44 @@ func init() {
 	})
 
 	// template-instance-controller
+	// we don't use cluster-admin, nor the cluster role OLM defines, but starting with 4.2 this controller needs to
+	// have the same authority as OLM for processing templates from the OpenShift console
+	// see https://github.com/operator-framework/operator-lifecycle-manager/blob/master/manifests/0000_50_olm_01-olm-operator.serviceaccount.yaml
+	// for what this change in role was based off of
 	addControllerRole(rbacv1.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + InfraTemplateInstanceControllerServiceAccountName},
 		Rules: []rbacv1.PolicyRule{
-			rbacv1helpers.NewRule("create").Groups(kAuthzGroup).Resources("subjectaccessreviews").RuleOrDie(),
-			rbacv1helpers.NewRule("update").Groups(templateGroup).Resources("templateinstances/status").RuleOrDie(),
+			rbacv1helpers.NewRule("*").Groups("*").Resources("*").RuleOrDie(),
 		},
 	})
 
 	// template-instance-controller
+	// the call to addControllerRole above also creates the necessary clusterrolebinding; however, in addition to that clusterrolebinding,
+	// for legacy purposes an admin role association is still needed for commands like oc adm policy remove-cluster-role-from-user admin ghost
+	// otherwise get error `error: unable to locate RoleBinding  for ClusterRole "admin"`
+	// TODO should that test be removed, or should the ClusterRoleBinding to "admin" be created in some other fashion for the cmd tests in test/cmd/policy.sh
```

Comment:
if the test was just testing removing the admin role from the TIC SA, it sounds like it's not a valid test any more.

Reply:
@bparees the tests that failed are https://github.com/openshift/origin/blob/master/test/cmd/policy.sh#L144-L145

AFAIK `ghost` is not some special case user identifier that means anybody.  At least I saw no indication of that in the post.

It definitely feels like a difference in initialization paths between the cmd tests and actual clusters.

If you like I'll add a preliminary commit that comments this stuff out and see where we land.

Reply:
since "ghost" is not a TIC SA, i don't see why that test is affected by these changes?  The "admin" role is still going to exist, and the ghost user doesn't exist, so i'd expect that test to continue to "pass" (report an error that the user does not exist) with or without your changes in this PR...

Reply:
We get this error message instead:

`error: unable to locate RoleBinding  for ClusterRole "admin"`

If I comment out:

```
	/*templateInstanceController := rbacv1helpers.NewClusterBinding(AdminRoleName).SAs(DefaultOpenShiftInfraNamespace, InfraTemplateInstanceControllerServiceAccountName).BindingOrDie()
	templateInstanceController.Name = saRolePrefix + InfraTemplateInstanceControllerServiceAccountName + ":admin"
	addDefaultMetadata(&templateInstanceController)
	controllerRoleBindings = append(controllerRoleBindings, templateInstanceController)*/

```

and 

```
	/*templateInstanceFinalizerController := rbacv1helpers.NewClusterBinding(AdminRoleName).SAs(DefaultOpenShiftInfraNamespace, InfraTemplateInstanceFinalizerControllerServiceAccountName).BindingOrDie()
	templateInstanceFinalizerController.Name = "system:openshift:controller:" + InfraTemplateInstanceFinalizerControllerServiceAccountName + ":admin"
	addDefaultMetadata(&templateInstanceFinalizerController)
	controllerRoleBindings = append(controllerRoleBindings, templateInstanceFinalizerController)*/
```

Reply:
When running `hack/test-cmd.sh policy.sh`

Reply:
/cc @openshift/sig-auth

Reply:
can you reproduce this in your own cluster?  (ie run the command, see the output, then remove the rolebinding from the SAs and run the command again?)

Reply:
I get the same result of `error: unable to locate RoleBinding  for ClusterRole "admin"`when deleting the binding of `admin` to the two template instance controller SAs.